ietf-optical-impairment-topology.yang

```diff
@@ -605,42 +635,316 @@ module ietf-optical-impairment-topology {
+        description 
+          "Polarization Mode Dispersion";
+      }
+      leaf roadm-cd {
+        type decimal64 {
+          fraction-digits 5;
+        }
+        units "ps/nm"; 
+        description "Chromatic Dispersion";
+      }            
+      leaf roadm-pdl {
+        type decimal64 {
+          fraction-digits 2;
+        }
+        units dB ;
+        description "Polarization dependent loss";        
+      }      
+      leaf roadm-inband_crosstalk {
+        type decimal64 {
+          fraction-digits 2;
+        }
+        units dB;
+        description
+          "In-band crosstalk, or coherent crosstalk, can occur in
+           components that can have multiple same wavelength inputs,
+           with the inputs either routed to different output ports,
+           or all but 1 blocked.
+           In the case of drop path it is the total of the ingress
+           to drop e.g. WSS and drop block crosstalk contributions.";
+      }
+      leaf roadm-maxloss {
+        type decimal64 {
+          fraction-digits 2;
+        }
+        units dB ;
+        description 
+          "The net loss from the ROADM input,to the output of the add/drop block.
+           If this path includes an amplifier, the amplifier gain reduces the net loss.  
+           This is before any additional drop path attenuation that may be required due to drop amplifier power contraints.
+           The max value correspond to worst case expected loss,including amplifier gain ripple or uncertainty";        
+      }
+      leaf roadm-minloss {
+        type decimal64 {
+          fraction-digits 2;
+        }
+        units dB ;
+        description 
+          "The net loss from the ROADM input,to the output of the add/drop block.
+           If this path includes an amplifier, the amplifier gain reduces the net loss.  
+           This is before any additional drop path attenuation that may be required due to drop amplifier power contraints.
+           The min value correspond to best case expected loss,including amplifier gain ripple or uncertainty";        
+      }
+      leaf roadm-typloss {
+        type decimal64 {
+          fraction-digits 2;
+        }
+        units dB ;
+        description 
+          "The net loss from the ROADM input,to the output of the add/drop block.
+           If this path includes an amplifier, the amplifier gain reduces the net loss.  
+           This is before any additional drop path attenuation that may be required due to drop amplifier power contraints.
+           The typ value correspond to typical case expected loss";        
+      }
+      leaf roadm-pmin {
+        type decimal64 {
+          fraction-digits 2;
+        }
+        units dBm ;
+        description 
+          "If the drop path has additional loss that is added, for example, to hit target power levels into a 
+           drop path amplifier,or simply,to reduce the power of a “strong” carrier(due to ripple,for example), 

```

Comment:
blank after ','
worst case -> may be change to min because there is no reason to say that a minimum value is a worst case (no good or evil there)

Reply:
ok